lib/usb-connection.js

```diff
@@ -23,7 +29,7 @@ try {
   // var VENDOR_REQ_IN  = usb.LIBUSB_REQUEST_TYPE_VENDOR | usb.LIBUSB_RECIPIENT_DEVICE | usb.LIBUSB_ENDPOINT_IN;
 } catch (e) {
   haveusb = false;
-  logs.err('WARNING: No usb controller found on this system.');
+  log.error('WARNING: No usb controller found on this system.');
 }
```

Comment:
Yea, I did something screwy with `nvm` when creating new bash sessions. The comment was mostly recognizing some missing error feedback around this part of logging. Could we also have a line that logs the error coming from the `try...catch` around `require('usb')`? Or would that be part of another patch?